README.rst

```diff
@@ -22,12 +22,16 @@ This repository includes shared utilities for:
 
 * `Plugin Infrastructure`_: Enables enhanced Django Plugin capabilities.
 
+* `Django User and Group Utilities`_: Includes user and group management utilities.
+
 .. _Cache Utilities: edx_django_utils/cache/README.rst
 
 .. _Monitoring Utilities: edx_django_utils/monitoring/README.rst
 
 .. _Plugin Infrastructure: edx_django_utils/plugins/README.rst
 
+.. _Django User and Group Management Utilities: edx_django_utils/user/README.rst
```

Comment:
Needs to match above. Drop “Management“.

Reply:
Done.